samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=3
+CONFIG_STACK_SENTINEL=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+
+# Use GPIO
+CONFIG_GPIO=y
+CONFIG_GPIO_NRFX=y
+CONFIG_GPIO_NRF_INIT_PRIORITY=40
```

Comment:
Why do you need a non-default priority?

Reply:
I'm not sure, either. It's copied from somewhere long time ago in previous project. Removed this line.